apps/spruce/src/hooks/useBreakingTask/index.ts

```diff
@@ -56,8 +56,9 @@ export const useBreakingTask = (taskId: string) => {
       },
     },
   });
-  // @ts-expect-error: FIXME. This comment was added by an automated script.
-  const task = getTaskFromMainlineCommitsQuery(breakingTaskData);
+  const task = breakingTaskData
+    ? getTaskFromMainlineCommitsQuery(breakingTaskData)
```

Comment:
Within this util function, do you think calling `reportError` is appropriate? It seems like if that were just removed, this code would function identically to what was deployed.

Reply:
Its not necessary but the schema implies the value can still be undefined since it is nullable. I can add an empty check to satisfy the typescript error but the `reportError` will likely never be called unless it actually encounters an empty value.

Reply:
Yeah, since null is a valid value as per the schema I could see it making sense to not treat that value as an error and just handle it instead. Either works though!